nb-adapters/adapter-cqld4/src/main/java/io/nosqlbench/adapter/cqld4/Cqld4DriverAdapter.java

```diff
@@ -29,10 +32,7 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-import java.util.ArrayList;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
This seems lazy to me when Idea will fill these in for you.

Reply:
There were some places where it didn't, and I put a wildcard in to resolve issues. I thought it would just clean them up.